### DISCUSSION TITLE
Laravel Sail SSL

### DISCUSSION THREAD

**Discussion by gentritabazi** [laravel/framework#36065] [Q&A]
Hello,

I'm using Laravel Sail on production, is any way how i can setup ssl (letsencrypt) ?

Thanks in advance.

**Comment by ahinkle** [comment]
Yes, that all depends on your server provider more than the Laravel application. Who is your server provider?

By taking the steps with your server provider, you can setup a SSL following an online guide. This wouldn't be much to do with the Laravel application in itself.
